lib/rules/lines-around-comment.js

```diff
@@ -93,6 +93,12 @@ module.exports = {
                     },
                     allowArrayEnd: {
                         type: "boolean"
+                    },
+                    ignorePattern: {
+                        type: "string"
+                    },
+                    applyDefaultPatterns: {
```

Comment:
Ok, I found some examples. I'm putting the deprecation in a different PR, though, so it gets a bit more visibility in the changelog.